backend/server/server_utils.py

```
@@ -239,7 +316,7 @@ async def handle_file_deletion(filename: str, DOC_PATH: str) -> JSONResponse:
 async def execute_multi_agents(manager) -> Any:
     websocket = manager.active_connections[0] if manager.active_connections else None
     if websocket:
-        report = await run_research_task("Is AI in a hype cycle?", websocket, stream_output)
+        report = await run_multi_agent_task("Is AI in a hype cycle?", websocket, stream_output)
```

Ruff (0.15.2)

[error] Undefined name `stream_output`

(F821)